llvm/lib/Target/RISCV/RISCVProcessors.td

```diff
@@ -266,11 +272,47 @@ def SIFIVE_P450 : RISCVProcessorModel<"sifive-p450", SiFiveP400Model,
                                        FeatureStdExtZfhmin,
                                        FeatureUnalignedScalarMem,
                                        FeatureUnalignedVectorMem],
-                                      [TuneNoDefaultUnroll,
-                                       TuneConditionalCompressedMoveFusion,
-                                       TuneLUIADDIFusion,
-                                       TuneAUIPCADDIFusion,
-                                       FeaturePostRAScheduler]>;
+                                      SiFiveP400TuneFeatures>;
+
+def SIFIVE_P470 : RISCVProcessorModel<"sifive-p470", SiFiveP400Model,
```

Comment:
The data sheet says P470 is RVA22 compliant. Can we use `!listconcat(RVA22S64Features, ...)` here? See the processor definition for `SPACEMIT_X60` below.

Reply:
What's the benefit of adding supervisor extensions? They aren't useable from applications. It just makes the ELF attribute very long.

Reply:
I've updated it to use a profile list, without the supervisor extensions.